trade-finance-manager-ui/templates/utilisation-reports/_macros/check-keying-data-table-row.njk

```diff
@@ -0,0 +1,56 @@
+{% import "./fee-record-status.njk" as feeRecordStatus %}
+{% import "../../_macros/table-cell-checkbox.njk" as tableCellCheckbox %}
+
+{% macro render(params) %}
+  {% set feeRecordPaymentGroup = params.feeRecordPaymentGroup %}
+
+  {% set feeRecords = feeRecordPaymentGroup.feeRecords %}
+  {% set paymentsReceived = feeRecordPaymentGroup.paymentsReceived %}
+  {% set status = feeRecordPaymentGroup.status %}
+  {% set displayStatus = feeRecordPaymentGroup.displayStatus %}
+
+  {% for feeRecord in feeRecords %}
+    {% set isFirstFeeRecordInGroup = loop.index === 1 %}
+    {% set isLastFeeRecordInGroup = loop.index === feeRecords.length %}
+
+    <tr
+      class="govuk-table__row"
+      data-cy="check-keying-data-table-row--feeRecordId-{{ feeRecord.id }}"
+    >
+      <th
+        scope="row"
+        class="govuk-table__header govuk-!-font-weight-regular{{ " no-border" if not isLastFeeRecordInGroup }}"
+      >
+        {{ feeRecord.facilityId }}
+      </th>
+      <td class="govuk-table__cell{{ " no-border" if not isLastFeeRecordInGroup }}">
+        {{ feeRecord.exporter }}
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric{{ " no-border" if not isLastFeeRecordInGroup }}">
+        {{ feeRecord.reportedFees }}
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric{{ " no-border" if not isLastFeeRecordInGroup }}">
+        {{ feeRecord.reportedPayments }}
+      </td>
+
+      <td class="govuk-table__cell govuk-table__cell--numeric{{ " no-border" if not isLastFeeRecordInGroup }}">
+        {% if isFirstFeeRecordInGroup and paymentsReceived %}
+          <ul class="payments-list">
+            {% for paymentReceived in paymentsReceived %}
+              <li class="payments-list-item">
+                {{ paymentReceived.formattedCurrencyAndAmount }}
+              </li>
+            {% endfor %}
+          </ul>
+        {% endif %}
+      </td>
+
+      <td class="govuk-table__cell{{ " no-border" if not isLastFeeRecordInGroup }}" data-sort-value="{{ displayStatus }}">
+        {{ feeRecordStatus.render({
+          status: status,
+          displayStatus: displayStatus
+        }) if isFirstFeeRecordInGroup else "" }}
+      </td>
+    </tr>
```

Comment:
I think the ticket is pretty clear that in the payments received bullet that we shouldn't be grouping fee records into multiple rows? Has there been a change to this that has not been recorded on the ticket?